docs/get-started.rst

```diff
@@ -138,7 +138,7 @@ On Debian / Ubuntu based systems::
 
 On RHEL-derivatives::
 
-    apt-get install libffi-devel openssl-devel python-devel python-virtualenv
+    dnf install libffi-devel openssl-devel python-devel python-virtualenv
```

Comment:
For those who don't know what DNF is:

> DNF was introduced in Fedora 18, and it has been the default package manager for Fedora since version 22
> — Wikipedia